examples/autoload.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+// require Requests autoloader or use the one provided by composer.
```

Comment:
```suggestion
// First, include the Requests Autoloader.
```

The code would be different if using the Composer autoload file as that automatically registers the autoloader, so let's not confuse people.

Reply:
I have changed this to your suggestion, do you also want me to change the `require_once` on the next line to a `include_once`?

I'm all for not confusing users, just an FYI: In my code I have preceded this with the Composer autoloader, and I'm not explicitly loading the Requests Autoloader.

Reply:
I think using `require_once` is appropriate here, so this should be fine.